pkgs/applications/networking/nextcloud-client/default.nix

```
@@ -29,15 +29,15 @@ stdenv.mkDerivation rec {
   # make sure an official release has been published on
   # https://github.com/nextcloud-releases/desktop/releases and re-check the
   # hash afterwards
-  version = "3.13.2";
+  version = "3.13.3";
 
   outputs = [ "out" "dev" ];
 
   src = fetchFromGitHub {
     owner = "nextcloud";
```

Is there a disadvantage to using
```suggestion
    owner = "nextcloud-releases";
```
?